### PR TITLE
Added camera input

### PR DESCRIPTION
**Pull request by ttanatb**
## Feature/Issue
camera now has input

## Implementation/Solution
some functions to help you orient your camera

### CODE REVIEW

File: Light Vox Engine/Engine.cpp [hunk 1]
```diff
@@ -253,11 +253,21 @@ void Engine::OnResize( uint32_t width, uint32_t height )
...
 {
     DEBUG_PRINT( "FPS: %f \n", 1.0f / time->GetDeltaFloatTime() );
-    printf( "FPS: %f \n", 1.0f / time->GetDeltaFloatTime() );
 }
 
 inline void Engine::Update()
 {
+    float dtfloat = time->GetDeltaFloatTime();
+    const static float speed = 2.f;
+    if ( inputManager->IsKeyDown( 0x41 ) ) // A
```
Comment (BenjaFriend) [inline Light Vox Engine/Engine.cpp]: Can't we use `VK_kEYCODE` for a bit more readable code?

File: Light Vox Engine/Graphics/Camera.cpp [hunk 2]
```diff
@@ -9,9 +9,11 @@ Camera::Camera()
...
-    position = glm::vec3( 0.f, 0.f, -2.f );
-    forward = glm::vec3( 0.0f, 0.f, 1.f );
+    position = glm::vec3( 0.f, 0.f, 0.f );
+    forward = glm::vec3( 0.0f, 0.f, -1.f );
     up = glm::vec3( 0.f, 1.f, 0.f );
+    right = glm::vec3( 1.0f, 0.0f, 0.0f );
+    pitchAngle = 0;
 }
 
 Camera::~Camera() {}
```
Comment (BenjaFriend) [inline Light Vox Engine/Graphics/Camera.cpp:19]: In consistent NameSpace usage here. In the CPP file we should be `using namespace XYZ` and then only have the class type in first part of the definition

**Review by BenjaFriend — CHANGES_REQUESTED** [review]
Overall good, I would just like to see those namespaces consistent

File: Light Vox Engine/Engine.cpp (hunk 1 above)
Comment (ttanatb) [reply Light Vox Engine/Engine.cpp]: there's no VK_KEYCODE for awsd -- i can re-map it to the arrow keys for now though

**Comment by ttanatb** [comment]
Ready for re-review

**Review by BenjaFriend — APPROVED** [review]

**Pull request merged**